apps/api/src/app/subscribers/subscribers.controller.ts

```diff
@@ -85,4 +92,37 @@ export class SubscribersController {
       })
     );
   }
+
+  @Get('/preferences')
+  @ExternalApiAccessible()
+  @UseGuards(JwtAuthGuard)
+  async getSubscriberPreference(@UserSession() user: IJwtPayload) {
+    const command = GetSubscriberPreferenceCommand.create({
+      organizationId: user.organizationId,
+      subscriberId: user._id,
+      environmentId: user.environmentId,
+    });
```

Comment:
Where is reference to the actual subscriberId? Looks like you are using the user._id here instead of a subscriberId.

I would suggest doing the following rest: `/subscribers/:subscriberId/preferences`

Reply:
Totally right I mismatched the ids here, and forgot to take the account of the subsbriber id!